libs/npm/core/src/hooks/modal/useCustomHeight.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+/*
+ * This hook is useful for retrieving the custom height of the browser for the modal popup
+ */
+export const useCustomHeight = () => {
```

Comment:
We already have this hook, it's called `useWindowSize`

Reply:
Updated, thanks